### ISSUE TITLE
Output repeats comment twice

### ISSUE THREAD

**Issue by tomviner** [timothycrosley/isort#304]
(X)Failing test here PR https://github.com/timothycrosley/isort/pull/303 showing that:

```python
from flask import url_for
# Whole line comment
from service import demo  # inline comment
from service import settings
```

becomes:

```python
from flask import url_for
# Whole line comment
from service import demo  # inline comment
# Whole line comment
from service import settings
```

**Comment by timothycrosley** [comment]
Thanks for identifying this issue so clearly, will make sure to get it fixed before 4.1.0

Thanks!

~Timothy

**Comment by timothycrosley** [comment]
Fixed in version 4.1.0

Thanks!

~Timothy
